spec/suites/layer/TooltipSpec.js

```diff
@@ -303,4 +303,18 @@ describe('Tooltip', function () {
 		layer.bindTooltip('Tooltip', {interactive: true});
 		layer.closeTooltip();
 	});
+
+	it("closes existent tooltip on new bindTooltip call", function () {
+		var layer = new L.Marker(center).addTo(map);
+		var eventSpy = sinon.spy();
+		layer.unbindTooltip = function () {
+			eventSpy();
+			L.Marker.prototype.unbindTooltip.call(layer);
+		};
```

Comment:
https://sinonjs.org/releases/v7.5.0/spies/#using-a-spy-to-wrap-an-existing-method